package.json

```diff
@@ -3,6 +3,10 @@
   "version": "3.0.0",
   "description": "Geospatial data visualization platform.",
   "license": "Apache-2.0",
+  "engines" : { 
+    "node" : ">= 4.0.0", 
```

Comment:
Are you sure this is correct?  We just saw problems on @chloeleichen's computer with 5.0.0, and I thought you had previously seen problems with 5.9.x even.  I know 5.10.x and 5.11.x work.